### ISSUE TITLE
issue with patsy 0.4.0  package in example `examples/notebooks/predict.ipynb` ?

### ISSUE THREAD

**Issue by stonebig** [statsmodels/statsmodels#2676]
I get an error at the 7th cell:
````
from statsmodels.formula.api import ols
data = {"x1" : x1, "y" : y}
res = ols("y ~ x1 + np.sin(x1) + I((x1-5)**2)", data=data).fit()
````
 
````
D:\WinPython\basedir27\build\winpython-2.7.10.amd64\python-2.7.10.amd64\lib\site-packages\patsy\design_info.pyc in __init__(self, factor, type, state, num_columns, categories)
     86         if self.type == "numerical":
     87             if not isinstance(num_columns, int):
---> 88                 raise ValueError("For numerical factors, num_columns "
     89                                  "must be an int")
     90             if categories is not None:

ValueError: For numerical factors, num_columns must be an int
````

**Comment by josef-pkt** [comment]
@stonebig Thanks for reporting

temporarily prio-high  because it doesn't look like a simple doc error
I have no idea what might be causing this, but I'm pretty sure we don't have test coverage for it.
My guess would be that it's a bug in patsy with being too strict on what's an `int`.

**Comment by huashuai** [comment]
I couldn't reproduce this with patsy 0.4.1, and pasty already updates to check again six.integer_types instead of just int.

**Comment by josef-pkt** [comment]
discussed on the mailing list, and fixed in patsy 0.4.1, closing

@huashuai  thanks for checking
